apps/web/src/server/lib/address-formatter.js

```diff
@@ -0,0 +1,23 @@
+import { join, map, pick } from 'lodash-es';
+
+/** @typedef {import('../applications/applications.types').ApplicationsAddress} ApplicationsAddress */
+
+/**
+ * converts a multi part address to a single string
+ *
+ * @param {ApplicationsAddress} address
+ * @returns {string}
+ */
+export const addressToString = (address) => {
+	let returnValue = '';
+
+	if (address) {
+		returnValue = join(
+			map(pick(address, ['addressLine1', 'addressLine2', 'town', 'postCode']), (value) => {
+				return value?.trim();
+			}),
+			', '
+		);
+	}
+	return returnValue;
```

Comment:
I don't think you need to define `returnValue = ''` here. If `address` is `undefined`, then the output of the function already becomes an empty string, which I believe is the intention here